cmake/Modules/FindTPLROCTHRUST.cmake

```diff
@@ -0,0 +1,2 @@
+FIND_PACKAGE(rocthrust REQUIRED)
```

Comment:
Do we have a way for users to say I don't have rocThrust, I understand I am potentially leaving performance on the table but I want configuration to proceed without error?
Would `CMAKE_DISABLE_FIND_PACKAGE_<PackageName>` work here?
Or is the intent that they explicitly pass `-DKokkos_ENABLE_ROCTHRUST=OFF`

Reply:
Yes, you should use `Kokkos_ENABLE_ROCTHRUST=OFF` to disable rocThrust.